util/git/repo/repo.go

```diff
@@ -68,7 +68,7 @@ func NewRepo(url string, creds git.Creds, insecure, enableLfs bool, disco func(r
 	if err != nil {
 		return nil, err
 	}
-	client, err := git.NewFactory().NewClient(url, workDir, creds, insecure, enableLfs, reporter)
+	client, err := git.NewClient(url, workDir, creds, insecure, enableLfs, reporter)
```

Comment:
intentional change?

Reply:
Yes, looks like factory is a left-over after some refactoring. It is not injected anywhere and not used in tests anymore.